tests/dualtor_mgmt/test_egress_drop_nvidia.py

```diff
@@ -0,0 +1,247 @@
+import time
+import logging
+import pytest
+import ptf.packet as scapy
+import json
+from tests.common.helpers.assertions import pytest_assert
+from tests.common.dualtor.mux_simulator_control import toggle_all_simulator_ports_to_upper_tor  # noqa F401
+from tests.common.dualtor.mux_simulator_control import toggle_simulator_port_to_lower_tor  # noqa F401
+from tests.common.dualtor.dual_tor_utils import \
+    upper_tor_host, lower_tor_host, dualtor_info, check_muxcable_status  # noqa F401
+from tests.common.fixtures.ptfhost_utils import change_mac_addresses, run_icmp_responder, run_garp_service  # noqa F401
+from tests.common.dualtor.mux_simulator_control import toggle_all_simulator_ports_to_rand_selected_tor # noqa F401
+from ptf.mask import Mask
+from ptf import testutils
+from tests.common.utilities import wait_until
+from ptf.testutils import simple_tcp_packet, simple_ipv4ip_packet
+from tests.common.plugins.allure_wrapper import allure_step_wrapper as allure
+
+pytestmark = [
+    pytest.mark.topology('dualtor')
```

Comment:
I suggest using topology mark `t0`, and check if the topology is `dualtor` in `skip_non_nvidia_platforms`.
It's because the test cases can be skipped unexpectedly if the topology is not set to `dualtor` when running the test.

Reply:
Hi Bing,
I can make the change, but I don't quite understand why we could run the "dualtor" topology dedicated test without setting the topology to "dualtor" in the pytest command?

Reply:
Hi @bingwang-ms, do you have a strong opinion on this? If yes, I will update it, thanks.

Reply:
Please keep the topology marker `dualtor`. I just checked the other dualtor test cases and found the `dualtor` marker is widely used.